lib/metrics.py

```diff
@@ -19,7 +19,7 @@
 def int_or_default(value, default=0):
     try:
         return int(value)
-    except Exception as e:
+    except Exception:
```

Comment:
The commit messages says that we are just updating libraries. But this is non-library code. If we are fixing linting issues, can we do this in a separate commit, please? (This goes for all the changes in this file)

Reply:
no longer in mr.

Reply:
You resolved this conversation but it does appear to actually still be in the MR.